### PR TITLE
Add FXIOS-5393 [v109] feature flag for creditcard autofill changes

### PR DESCRIPTION
**Pull request by nbhasin2**
**[Jira 5393
](https://mozilla-hub.atlassian.net/browse/FXIOS-5393)**

### CODE REVIEW

File: nimbus.fml.yaml [hunk 1]
```diff
@@ -508,6 +508,22 @@ features:
             "move-actions": true,
             "toolbar-changes": true
         }
+  creditcard-autofill:
+    description: This property defines whether credit card autofill is enabled / disabled
+    variables:
+      creditcard-changes:
```
Comment (adudenamedruby) [inline nimbus.fml.yaml:514]: Would this be better named as `status`, as it's the status of the feature? `creditcard-changes` isn`t very clear about what exactly it is, and we've used `status` in a few other features as the enabled/disabled key. I should probably document this somewhere, really.

File: nimbus.fml.yaml [hunk 2]
```diff
@@ -508,6 +508,22 @@ features:
             "move-actions": true,
             "toolbar-changes": true
         }
+  creditcard-autofill:
```
Comment (adudenamedruby) [inline nimbus.fml.yaml:511]: I would call this `credit-card-autofill` as credit card is, technically, two words.

File: nimbus.fml.yaml [hunk 3]
```diff
@@ -508,6 +508,22 @@ features:
...
+    description: This property defines whether credit card autofill is enabled / disabled
```
Comment (adudenamedruby) [inline nimbus.fml.yaml:512]: I would argue that the top level feature shouldn't be the on/off status, that's the variable. The top level nimbus feature should always be forward looking. So something like `This property defines the credit card autofill feature`

File: nimbus.fml.yaml (hunk 1 above)
Comment (nbhasin2) [reply nimbus.fml.yaml:514]: good call

**Review by adudenamedruby — APPROVED** [review]

**Review by yoanarios — APPROVED** [review]

**Comment by nbhasin2** [comment]
The test which is failing has nothing to with this PR
I ran it locally and it passed

image

Will be merging this in.

**Pull request merged**